ding2.make

```diff
@@ -356,7 +356,7 @@ libraries[ckeditor][directory_name] = "ckeditor"
 libraries[ckeditor][destination] = "libraries"
 
 libraries[leaflet][download][type] = "get"
-libraries[leaflet][download][url] = "http://leaflet-cdn.s3.amazonaws.com/build/leaflet-0.7.2.zip"
+libraries[leaflet][download][url] = "http://cdn.leafletjs.com/downloads/leaflet-0.7.3.zip"
```

Comment:
Leaflet 0.7.2 is no longer available and if I did not include this then the build broke as drush make failed.

[This has already been fixed in the consolidated codebase](a841869028bb0f1eb4b7a4f891c996f9159d23f0) so I have removed the commit from this PR.